### PR TITLE
Ubuntu 16.04 instructions, fixes #81

### LINKED ISSUE #81: Installation on Ubuntu 16.04 doesn't work
After having installed it with apt-get, I run:

```
# pghero config:set DATABASE_URL=postgresql://myuser:mypwd@localhost:5432/mydb
```

```
# pghero scale web=1
Scaling up...
/usr/bin/pghero: line 155: initctl: command not found
```
I don't know what `initctl` is. Not something I can install:
```
# apt-cache search initctl
```
(yields no results)

It seems I can run...:
```
# pghero scale
```
without any errors. But nothing gets started.

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

File: guides/Linux.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Packaged for:
 
+- Ubuntu 16.04 (Trusty)
```
Comment (ankane) [inline guides/Linux.md]: This should be Xenial.

File: guides/Linux.md [hunk 2]
```diff
@@ -15,11 +16,11 @@ Packaged for:
 
 ## Installation
 
-Ubuntu 14.04 (Trusty)
+Ubuntu 16.04 (Xenial)
```
Comment (ankane) [inline guides/Linux.md]: Can we add this instead of replacing so we still have the 14.04 instructions?

**Comment by ankane** [comment]
I went ahead and pushed updated instructions. Thanks @peterbe

**Comment by peterbe** [comment]
Yay!

**Pull request closed without merging**